apis/Google.LongRunning/Google.LongRunning/Operation.cs

```diff
@@ -202,14 +204,15 @@ private void MaybeFireCallback(Action<TMetadata> metadataCallback)
         {
```

Comment:
Hmm. Not sure. Maybe stick with the sync version and add an overload later if we get requests for that? Note that we won't have a synchronization context (it could be called from any thread) so at least some of the normal benefits of async aren't present anyway...